src/modules/datepicker/directives/datepicker.directive.ts

```diff
@@ -148,6 +151,18 @@ export class SuiDatepickerDirective
         }
     }
 
+    public ngOnInit():void {
```

Comment:
The thing is, as I imagined it, that if `[pickerInitialDate]` was set, that meant that the input should be pre populated (i.e. `this._selectedDate` set) with the given date, or today() if null was passed, or do nothing (i.e. same as previous behavior) if `[pickerInitialDate]` was not used or was undefined.

Your proposal only concerns with the date the user will see when opening the pop up. Which is also interesting. Which way to go? 

I personally think that if we allow for an `pickerInitialDate` that date should be selected. But it is just preference/semantics.